service/db/models.py

```diff
@@ -76,7 +76,8 @@
     "league_score_journal",
     metadata,
     Column("id", Integer, primary_key=True),
-    Column("login_id", Integer, ForeignKey("login.id")),
```

Comment:
the foreign key is not mentioned in the actual sql migrations. I guess this here was a mistake then? If someone with more experience could comment on this, that would be great

Reply:
Hmm, is the league database totally separate from the main FAF database? In that case I guess it's possible to end up with values in these fields that don't actually exist in the main FAF database since there's no integrity checking on the database side. As far as the SQLAlchemy definitions are concerned I think it uses the `ForeignKey` during table creation (which we don't do on the lobby server side since it is set up to read only) and for generating the correct 'on' clauses automatically when joining tables (which wouldn't apply here since there is no `login` model defined.

That is to say, it's probably fine to remove... maybe?

Reply:
Afaik the databases are separate, but I don't know if there is a way to do some integrity checking between them. Maybe @Brutus5000 can shed some light on this.